selfdrive/hardware/base.py

```diff
@@ -105,3 +105,7 @@ def get_thermal_config(self):
   @abstractmethod
   def set_screen_brightness(self, percentage):
     pass
+
+  @abstractmethod
+  def get_cloudlog_enabled(self):
+    pass
```

Comment:
I would just put this in athenad, don't think it belongs in the HW abstraction layer